packages/cli/src/commands/test.js

```diff
@@ -10,7 +11,7 @@ const jest = require('jest')
 // TODO: Get from redwood.toml
 const sides = getProject().sides
 
-// https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/scripts/test.js#L39
+// https://github.com/facebook/create-react-app/blob/cbad256a4aacfc3084be7ccf91aad87899c63564/packages/react-scripts/scripts/test.js#L39
```

Comment:
This actually has nothing to do with this PR, but it felt silly to create a separate PR just for this. At least I made a separate commit for it...